packages/dialtone-vue2/components/illustration/illustration.mdx

```diff
@@ -0,0 +1,37 @@
+import { Canvas, Story, Subtitle, Controls, Meta } from '@storybook/blocks';
+
+import * as IllustrationStories from './illustration.stories';
+
+<Meta of={IllustrationStories}/>
+
+# Illustration
+
+<Subtitle>
+  The Illustration component provides a set of images to provide context to your application.
+</Subtitle>
+
+## Base Style
+
+<Canvas>
+  <Story of={IllustrationStories.Default} />
+</Canvas>
+
+## Slots, Props & Events
+
+<Controls />
+
+## Usage
+
+### Import
+
+In order for dt-icon to be tree-shakeable, import it as following, depending on the bundler your application is using.
```

Comment:
Remove if not needed or update accordingly

```suggestion
```

Reply:
removed